tools/cross-compile.sh

```diff
@@ -106,6 +106,10 @@ echo "Windows 64"
 GOOS=windows GOARCH=amd64 make build
 create_artefact_windows "win64"
 
+echo "Windows ARM64"
+GOOS=windows GOARCH=arm64 make build
+create_artefact_windows "winarm64"
```

Comment:
nit: `win-arm64`? (Just for consistency with other target names: `linux-arm64`, `macos-arm64`)